test/mlir/onnx/parse/test_abs.json.mlir

```diff
@@ -0,0 +1,78 @@
+// RUN: sed 's/^[ \t]*\/\/.*//' %s | onnx-mlir --EmitONNXBasic --printIR | FileCheck %s
```

Comment:
the sed command to remove `//` comments is rather cryptic

see comment in `FrontendDialectTransformer.cpp` about changing onnx-mlir to ignore comments

alternatively we could make the RUN line more self explanatory by calling a `strip-comments` script that runs the sed command, but I'm not sure how to put the script in a place where the lit test runner can find it

Reply:
> see comment in FrontendDialectTransformer.cpp about changing onnx-mlir to ignore comments

I like that. So comments are ignored for the mlir textual format (which it already does, from my understanding and experience) as well as now from the JSON file? 
I am glad someone will not have errors because of a badly formed cryptic sed command (love sed, but it may not be a generally shared feeling :-) )